### ISSUE TITLE
Minitest 5 verbose settings are printing a line per test

### ISSUE THREAD

**Issue by zmoazeni** [tmm1/test-queue#22]
tl;dr `TEST_QUEUE_VERBOSE` output is different between Minitest 4 and Minitest 5. Minitest 5 outputs a line per each test within all test files.

Here are two simple files and the output to illustrate what I'm talking about:

``` ruby
# in example_minitest4_test.rb

require "minitest/autorun"

class Meme
  def i_can_has_cheezburger?
    "OHAI!"
  end

  def will_it_blend?
    "YES!"
  end
end

class TestMeme < Minitest::Unit::TestCase
  def setup
    @meme = Meme.new
  end

  def test_that_kitty_can_eat
    assert_equal "OHAI!", @meme.i_can_has_cheezburger?
  end

  def test_that_it_will_not_blend
    refute_match /^no/i, @meme.will_it_blend?
  end

  def test_that_will_be_skipped
    skip "test this later"
  end
end


# in example_minitest5_test.rb
require "minitest/autorun"

class Meme
  def i_can_has_cheezburger?
    "OHAI!"
  end

  def will_it_blend?
    "YES!"
  end
end

class TestMeme < Minitest::Test
  def setup
    @meme = Meme.new
  end

  def test_that_kitty_can_eat
    assert_equal "OHAI!", @meme.i_can_has_cheezburger?
  end

  def test_that_it_will_not_blend
    refute_match /^no/i, @meme.will_it_blend?
  end

  def test_that_will_be_skipped
    skip "test this later"
  end
end

# in a Gemfile

source 'https://rubygems.org'

# gem 'minitest', '4.7.5'
gem 'minitest', '5.5.1' # the latest as of this date
gem 'test-queue'
```

Running each file with with their respective minitest versions and `TEST_QUEUE_VERBOSE` gives different results:

```
~/projects/tmp/working_with_test_queue ∴ TEST_QUEUE_WORKERS=1 TEST_QUEUE_VERBOSE=true be minitest-queue ./example_minitest4_test.rb
Starting test-queue master (/tmp/test_queue_14882_70290198415720.sock)

==> Starting test-queue worker [1] (14883 on aybara.local) - iterating over /tmp/test_queue_14882_70290198415720.sock

    TestMeme: ..S  <0.002>


Finished tests in 0.002478s, 1210.6538 tests/s, 1210.6538 assertions/s.

3 tests, 3 assertions, 0 failures, 0 errors, 1 skips


==> Summary (1 workers in 0.0084s)

    [ 1]         3 tests, 3 assertions, 0 failures, 0 errors, 1 skips         1 suites in 0.0062s      (pid 14883 exit 0)

~/projects/tmp/working_with_test_queue ∴ TEST_QUEUE_WORKERS=1 TEST_QUEUE_VERBOSE=true be minitest-queue ./example_minitest5_test.rb
Starting test-queue master (/tmp/test_queue_14825_70299945470700.sock)

==> Starting test-queue worker [1] (14826 on aybara.local) - iterating over /tmp/test_queue_14825_70299945470700.sock

Run options: --seed 16398

# Running:

    TestMeme: .  <0.000>
    TestMeme: .  <0.000>
    TestMeme: S  <0.000>


Finished in 0.002102s, 1427.2122 runs/s, 1427.2122 assertions/s.

3 runs, 3 assertions, 0 failures, 0 errors, 1 skips

You have skipped tests. Run with --verbose for details.

==> Summary (1 workers in 0.0092s)

    [ 1]          3 runs, 3 assertions, 0 failures, 0 errors, 1 skips         1 suites in 0.0076s      (pid 14826 exit 0)
```

(If it's hard to spot, each individual test within a test file has a dot and a runtime with Minitest 5)

I think it stems from a change in Minitest version 5.0.4 where ["Refactored Reporter into ProgressReporter and SummaryReporter. (idea: phiggins, code:me+scotch)"](https://github.com/seattlerb/minitest/blob/master/History.txt#L176)

I haven't been able to find a fix yet, but wanted to post a reproducible issue in the meantime. `TEST_QUEUE_VERBOSE` was super handy to see which tests took the longest.

**Comment by zmoazeni** [comment]
Oh, I guess this isn't just about `TEST_QUEUE_VERBOSE`. The worker's output will be printed out in the event there is a error in one of the tests the worker executes.

It will ignore the output of the successful workers.
